rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/rascal/RascalLanguageServices.java

```diff
@@ -100,6 +107,8 @@ public RascalLanguageServices(RascalTextDocumentService docService, BaseWorkspac
         documentSymbolEvaluator = makeFutureEvaluator(exec, docService, workspaceService, client, "Rascal document symbols", monitor, null, false, "lang::rascal::lsp::DocumentSymbols");
         semanticEvaluator = makeFutureEvaluator(exec, docService, workspaceService, client, "Rascal semantics", monitor, null, true, "lang::rascalcore::check::Summary", "lang::rascal::lsp::refactor::Rename");
         compilerEvaluator = makeFutureEvaluator(exec, docService, workspaceService, client, "Rascal compiler", monitor, null, true, "lang::rascalcore::check::Checker");
+        actionEvaluator = makeFutureEvaluator(exec, docService, workspaceService, client, "Rascal actions", monitor, null, true, "lang::rascal::lsp::Actions");
```

Comment:
I'm not a fan of yet another evalutor getting added, this creates a bigger memory pressure. It also slows down initial startup, as they are all loaded in parallel when VS Code starts. 

Can't this one go into the `semanticEvaluator`? I mean, it's a balance, since `Summary` and especially `Rename` can also take quite a while, so it would mean the actions waiting in the queue.

Reply:
I think that would be acceptible. Currently my machine is only a bit sooner to present the actions than to present the summary. And you are right there will be a large overlap between summary code and actions code eventually anyway, so leading to memory duplication that is unwarranted.